packages/@okta/vuepress-site/docs/guides/common-hook-set-up-steps/adding-basic-auth/index.md

```diff
@@ -2,20 +2,24 @@
 title: Adding Basic Authorization and Body Parsing
 ---
 
-The Glitch projects do not have any authorization or body parsing code by default. To include this content:
+The default, new Glitch projects do not have any authorization or body parsing code. To include this content:
```

Comment:
Yes, a little clunky. I've re-worded as follows to use the Glitch terminology, rather than throwing in "default":

"The Glitch project templates do not have ...."